gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-8.13-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.12.1-bin.zip
```

Comment:
why do we downgrade gradle here?

Reply:
Because of https://github.com/MaSch0212/deno-gradle-plugin/issues/4 :(